src/EventStore.Projections.Core.Tests/ClientAPI/Cluster/specification_with_standard_projections_runnning.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
```

Comment:
Rebasing should resolve this issue as there was a [PR that resolved this already](https://github.com/EventStore/EventStore/pull/2076).